praw/models/reddit/mixins/inboxable.py

```diff
@@ -25,6 +50,18 @@ def mark_read(self):
         .. note:: This method pertains only to objects which were retrieved via
                   the inbox.
 
+        Example usage:
+
+        .. code:: python
+
+           inbox = reddit.inbox.unread()
+
+           # mark all unread inbox items as read
```

Comment:
While this definitely is an example, marking all messages as read can be done more efficiently by passing multiple messages to `inbox.mark_read`. Perhaps we can add a fictitious `do_work(message)` call in the loop to indicate we're processing all unread messages.